packages/@aws-cdk/aws-apigateway/lib/domain-name.ts

```diff
@@ -38,15 +39,15 @@ export interface DomainNameOptions {
   /**
    * The Transport Layer Security (TLS) version + cipher suite for this domain name.
    * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-domainname.html
-   * @default SecurityPolicy.TLS_1_0
+   * @default - CloudFormation will default to TLS_1_0
```

Comment:
Why change this? This is how we transmit both defaults the CDK chooses and CloudFormation. As far as a customer is concerned, the Domain Name will be configured with TLS 1.0. Whether it's configured by CDK or CloudFormation is irrelevant.

Could you revert this back to its original content?

Reply:
Yep.